zips/draft-jackgavigan-defer-and-extend.rst

```diff
@@ -0,0 +1,186 @@
+::
+
+    ZIP: XXX
+    Title: Defer Lockbox Distribution and Extend the Current Dev Fund
+    Owners: Jack Gavigan <jackgaviganzip@gmail.com>
+    Status: Draft
+    Category: Consensus / Process
+    Created: 2025-03-31
+    License: MIT
+    Pull-Request: <https://github.com/zcash/zips/pull/???>
+
+
+Terminology
+===========
+
+The key words "SHALL NOT", and "SHOULD" in this document are to be interpreted as 
+described in BCP 14 [#BCP14]_ when, and only when, they appear in all 
+capitals.
+
+"Zcash Community Advisory Panel", also called "ZCAP", refers to the panel of
+community members assembled by the Zcash Foundation and described at [#zcap]_.
+
+"Zcash Shielded Assets", also called "ZSAs", refers to the protocol changes
+described in draft ZIP 226 [#zip-0226]_ and draft ZIP 227. [#zip-0227]_ 
+
+The character § is used when referring to sections of the Zcash Protocol
+Specification. [#protocol]_
+
+The term "Mainnet" is to be interpreted as described in § 3.12 ‘Mainnet and 
+Testnet’. [#protocol-networks]_
+
+"Lockbox" refers to the in-protocol lockbox described in ZIP 1015, [#zip-1015]_ 
+and specified in ZIP 2001. [#zip-2001]_
+
+"Current Dev Fund" refers to the block subsidy allocation described in ZIP 1015, 
+[#zip-1015]_ and specified in ZIP 2001. [#zip-2001]_
+
+"Zcash Community Grants", also called "ZCG", refers to the committee described 
+in [#zip-1015]_.
+
+"Bootstrap Project", also called "BP", refers to the 501(c)(3) nonprofit 
+corporation of that name.
+
+"Zcash Foundation", also called "ZF", refers to the 501(c)(3) nonprofit 
+corporation of that name.
+
+"Electric Coin Company", also called "ECC", refers to the Zerocoin Electric 
+Coin Company, LLC.
+
+
+Abstract
+========
+This ZIP proposes expressly prohibiting any disbursement of funds from the 
+Lockbox until after Zcash Shielded Assets is active on Mainnet, and extending 
+the Current Dev Fund for one year. 
+
+
+Motivation
+==========
+
+Zcash Shielded Assets (ZSAs) is arguably the single most impactful change to the 
+Zcash protocol since the launch of Zcash in late 2016. There is clear consensus
+support for ZSAs across the Zcash community, and a ZCAP poll carried out in 
+August 2021 indicated that the Zcash community believed that ZSAs should be 
+first priority for Zcash following the activation of NU5. [#priorities]_
+
+In January 2022, ZCG provided funding for QEDIT to research and implement ZSAs 
+[#zsagrant]_, and the functionality was demonstrated using a fork of zcashd 
+during Zcon4 in August 2023. [#zsademo]
+
+Due to ECC's decision to no longer support new protocol features in the zcashd 
+node implementation, zcashd must be deprecated before ZSAs can activate on 
+Mainnet. Zcashd deprecation was expected to be complete by April 2025. However, 
+the necessary work was not prioritized. [#nu7-timeline]_ As a result, as of March 
```

Comment:
@jackgavigan please do not spread outright falsehoods.

The timeline that was originally given was the most optimistic possible timeline, and not a commitment by any organization. Furthermore, due to the complexity of reimplementing the zcashd wallet, even if no other efforts had been undertaken this work would not have been complete by April 2025. Your expectation of this completion date was your own, not a realistic schedule based on an accurate assessment of the task.

Reply:
@jackgavigan I agree with @nuttycom that this needs to be rephrased, I think the sentiment is there - we've (in the ecosystem as a whole, including ZF and myself personally) have blundered repeatedly since zcon4 leading to delays - but at the time that the timeline you're referring to was released, it was clear that April was the best possible timeline if the stars aligned, not a worst-case hard deadline.

Reply:
@nuttycom For the record, I do **not** appreciate being falsely accused of spreading falsehoods. 

> Your expectation of this completion date was your own...

No, it was created by Josh Swihart stating, during the [Arborist Call dedicated to discussing ZSAs, NU6 and NU7 on 18th April 2023](https://youtu.be/C_S5e2WNIe8?si=Bs8i62YpK7Yp-T3O&t=2604), that zcashd deprecation would be complete "well before" April 2025. 

Others interpreted Josh's words [in exactly the same way](https://forum.zcashcommunity.com/t/nu7-timeline-and-the-dev-fund-expiration/50134/23).

@arya2 When you say _"at the time that the timeline you're referring to was released"_, what timeline do you think I'm referring to?

Reply:
> When you say "at the time that the timeline you're referring to was released", what timeline do you think I'm referring to?

I was thinking of this one: 
![image](https://github.com/user-attachments/assets/10ac5604-d5f2-462a-b122-f53916d518bd)

Reply:
> Others interpreted Josh's words [in exactly the same way](https://forum.zcashcommunity.com/t/nu7-timeline-and-the-dev-fund-expiration/50134/23).

These timelines were set by determining the *earliest possible* timeline along which NU7 could be released, given the lead times required for node deprecation. This was used to compute backward to determine a ZIP submission deadline. That's all.